yarn.lock

```diff
@@ -24809,6 +24828,15 @@ terser@^4.1.2, terser@^4.3.9, terser@^4.6.12, terser@^4.6.2, terser@^4.6.3:
     source-map "~0.6.1"
     source-map-support "~0.5.12"
 
+terser@^5.7.2:
```

Comment:
is it possible to dedupe terser version ? it's being duped because of pinned `5.7.1` version